src/connect/index.js

```diff
@@ -35,6 +35,7 @@ function connect(connectOptions, callback) {
       maxReceivedFrameSize: constants.MAX_WEBSOCKET_FRAME_SIZE,
       maxReceivedMessageSize: constants.MAX_WEBSOCKET_FRAME_SIZE,
     },
+    propogationDelayWaitMillis: connectOptions.propogationDelayWaitMillis,
```

Comment:
Nit: `propagation` is misspelled

Reply:
It's been copy-pasta'd all the way from ethrpc! 

https://github.com/ethereumjs/ethrpc/commit/b04d64225b657c628d57d7df818b2ea11e0fd47e#diff-0392e5d28b9d90768af39a89a5f0198fR19

i'll fix